preset.js

```diff
@@ -36,4 +36,11 @@ module.exports = {
       resolveLoader: resolvedConfig.resolveLoader,
     };
   },
+  webpackFinal: config => ({
+    ...config,
+    module: {
+      ...config.module,
+      rules: config.module.rules.slice(0, -3),
```

Comment:
Could you add a comment why you slice off here? what rules are removed by that?

Reply:
Vue project's webpack config has css loaders and stuff. But storybooks adds them again which creates an error. This part of code is making sure that it doesn't happen.

Reply:
Do you remove the Vue ones or the storybook ones? just comment the code, so others know whats goin on here, makes it alot easier to debug problems with missing rules/unexpected rulesets.

Reply:
Storybook ones

Reply:
https://github.com/storybooks/vue-cli-plugin-storybook/commit/75e9d8a57b0c80fb63f82c616d24e8a12d66aa47